<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderWizardTargetPage</class>
 <widget class="QWidget" name="FolderWizardTargetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="subHeader">
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: #e20074</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="content">
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: #191919</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subContent">
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: #191919</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="folderTreeWidget">
       <property name="styleSheet">
        <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: #191919</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Folders</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="styleSheet">
          <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: #191919</string>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addFolderButton">
         <property name="styleSheet">
          <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: #191919</string>
         </property>
         <property name="text">
          <string>Create folder</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="folderEntry">
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: #191919;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warnLabel">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;Segoe UI&quot;;
color: red</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
